src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable drive. 
+
+## Creating the Bootable Drive
+This section will teach you how to create a bootable drive for your needs.
```

Comment:
This kinda feels like a duplicate of line 2. Opinions on this?

Reply:
Done